pkg/elementalConfig/config_test.go

```diff
@@ -212,9 +214,12 @@ var _ = Describe("Types", Label("types", "config"), func() {
 						switch cmd {
 						case "cat":
 							return []byte(constants.SystemLabel), nil
-						default:
-							return []byte{}, nil
+						case "lsblk":
```

Comment:
even the full switch feels duplicated a bit

Reply:
This is true, but these switch statement are expected shell commands. They may look the same now but at some point, some Go command might introduce a shell command that will make these mock different. Then we will need to switch back to separate mocking per test. I'm not sure it's worth DRYing. After all, while making changes to a test, we should rather not worry that we may break the mocks for other tests.